_pytest/assertion/rewrite.py

```diff
@@ -128,7 +128,7 @@ def find_module(self, name, path=None):
                     # One of the path components was not a directory, likely
                     # because we're in a zip file.
                     write = False
-                elif e in [errno.EACCES, errno.EROFS]:
+                elif e in [errno.EACCES, errno.EROFS, errno.EPERM]:
```

Comment:
we should probably make that a global tuple while we are at it - i think i need the same in cache-provider as well